File: sdk/python/examples/10_credits.py
"""10_credits:余额 / 账本 / 兑换。"""
from helpers import client

combee = client()
print("balance:", combee.credits.balance())
print("transactions:", combee.credits.transactions(10)["items"])
# 兑换(需要 admin 先生成 voucher):combee.credits.redeem("CMB-XXXX-XXXX-XXXX")

File: sdk/typescript/examples/10_credits.mjs
// 10_credits:余额 / 账本 / 兑换。
import { client } from "./00_helpers.mjs";
const combee = client();
console.log("balance:", await combee.credits.balance());
console.log("transactions:", (await combee.credits.transactions(10)).items);
// 兑换(需要 admin 先生成 voucher):await combee.credits.redeem("CMB-XXXX-XXXX-XXXX");
